content/applications/sales/sales/amazon_connector/features.rst

```diff
@@ -1,28 +1,29 @@
 =========================
-Amazon Connector Features
+Amazon Connector features
 =========================
 
-The **Amazon Connector** synchronizes the orders between Amazon and your Odoo database, which
-reduces considerably the amount of time spent on your Amazon Seller Central dashboard, making your
-daily routine a lot easier.
+The *Amazon Connector* synchronizes orders between Amazon and Odoo, which considerably reduces the
+amount of time spent on the Amazon Seller Central dashboard.
 
-Supported Features
+Supported features
 ==================
 
-The connector is able to:
+The *Amazon Connector* is able to:
 
-- Synchronize (Amazon to Odoo) all confirmed orders (both FBA and FBM) and their order items which
+- Synchronize (Amazon to Odoo) all confirmed orders (both FBA and FBM), and their order items, which
   include:
 
-  - the product’s name, description and quantity
-  - the shipping costs for the product
-  - the gift wrapping charges
+  - product name, description, and quantity
+  - shipping costs for the product
+  - gift wrapping charges
 
-- Create on Odoo any missing partner related to an order (contact types supported: contact and
+- Create any missing partner related to an order in Odoo (contact types supported: contact and
   delivery).
```

Comment:
Yes, it's my understanding (based on the previous doc's context) that this does, indeed, refer to the delivery address. I'll make a quick clarification change now to avoid any confusion. Thanks for pointing this out. 👍